core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/keymanager/delete/KeystoreFileManager.java

```diff
@@ -39,6 +41,9 @@
 public class KeystoreFileManager {
```

Comment:
Suggestion: Now this is used by the import flow, should it be pulled up a package from `delete` to `keymanager`?

Reply:
I didn't realize it was in delete subpackage. Will move it.